### DISCUSSION TITLE
Does Harbor support Brute force ?

### DISCUSSION THREAD

**Discussion by ramkrish12** [goharbor/harbor#19427] [Q&A]
Hello Harbor Community Experts, 

Can you please confirm if Harbor is enabled / configurable for brute force protection. 

please help understand the same. 

Regards,
Ram.

**Comment by stonezdj** [comment]
Yes， to avoid brute force, Harbor will lock the account for 1.5s if the password is wrong.

**Comment by ramkrish12** [comment]
HI @stonezdj, 

Thanks for the response. few more question to clarity. 
1. for the Bruteforce, whether lock period is configurable ? 
2. Whether Harbor also support blacklisting ? 

Regards,
Ram
